lib/Models/Catalog/createCatalogItemFromFileOrUrl.ts

```diff
@@ -32,28 +31,21 @@ export default function createCatalogItemFromFileOrUrl(
   if (dataType === "auto") {
     return createUrlReferenceFromUrl(name, terria, isUrl).then(newItem => {
       if (!isDefined(newItem)) {
-        return tryConversionService(name, terria, viewState, confirmConversion);
-      } else {
-        // It's a file or service we support directly
-        // In some cases (web services), the item will already have been loaded by this point.
-        return loadItem(newItem, fileOrUrl);
+        terria.raiseErrorToUser(
```

Comment:
Not sure if this is a problem or not, but if I upload a random unsupported file (eg. a random PNG) I get "Failed to create reference" rather than a useful error message. I think this issue exists in `main` already though, so I won't block the PR.

Reply:
Here I made an issue: https://github.com/TerriaJS/terriajs/issues/6197

Reply:
Yeah, the UrlReference errors aren't great at the moment.

I'm not sure if it was possible to get the conversion prompt/error in `main` (I tried a bit, but only got UrlReference errors), but after this PR you won't be able to get even more misleading errors about a conversion service.